R/lfe-tidiers.R

```diff
@@ -5,9 +5,11 @@
 #' @template param_confint
 #' @param fe Logical indicating whether or not to include estimates of
 #'   fixed effects. Defaults to `FALSE`.
-#' @param robust Logical indicating robust or clustered SEs should be used.
-#'   Following lfe::summary.felm, it defaults to `FALSE` when there is no
-#'   clustering variable, and defaults to `TRUE` when there is one. 
+#' @param se.type Character indicating the type of standard errors. Defaults to
```

Comment:
List values directly, e.g. `One of "robust", "cluster", "iid", or "default"`?

Reply:
Done.